pages/_app.jsx

```diff
@@ -1,13 +1,35 @@
-import { useContext } from 'react'
+import { useEffect } from 'react'
 import '../styles/index.css'
 import "cropperjs/dist/cropper.css";
 import { AuthenticationProvider } from 'swrlit'
 import Head from 'next/head'
+import { useRouter } from 'next/router'
 import { DndProvider } from 'react-dnd'
 import { HTML5Backend } from 'react-dnd-html5-backend'
 import { SWRConfig } from 'swr'
+import * as Fathom from 'fathom-client'
 
 function MyApp({ Component, pageProps }) {
+  const router = useRouter()
+
+  useEffect(() => {
+    // Initialize Fathom when the app loads
+    Fathom.load('TMUOYFGA', {
```

Comment:
should TMU0YFGA go in an environment variable?

Reply:
Probably eventually yeah!